packages/nimble-components/src/utilities/tests/parameterized.ts

```diff
@@ -1,23 +1,7 @@
 // eslint-disable-next-line no-restricted-globals
```

Comment:
This is in `src/utilities/tests`, whose README says is for nimble-components internal test utilites. We also have `src/testing`, whose README says is for things that could also be used by clients. Do you think this belongs in the latter place (either now or eventually once it's had some runtime)?

Reply:
I think once it has some runtime we should move it to a new package in the styleguide repo to share across applications
I think that's captured sufficiently here: https://github.com/ni/javascript-styleguide/issues/125#issuecomment-1631682110